pdf/README.md

```diff
@@ -0,0 +1,29 @@
+# PDF Generation of Toolkit
+
+The contents of this folder are used towards the automated generation of a PDF of the Digital Public Goods Toolkit. This is accomplished through the preprocessing and subsequent collating of selected Markdown files to output a single PDF file.
+
+## Requirements
+
+Generating a PDF, requires both `pandoc` (v2.9 or higher) and `TeX` to be installed in the system.
```

Comment:
No need for a comma in this sentence, FWIW.

Reply:
Thanks, corrected in fc80119